feathr_project/registry/main.py

```diff
@@ -0,0 +1,77 @@
+import os
+from typing import Optional
+from fastapi import APIRouter, FastAPI, HTTPException
+from starlette.middleware.cors import CORSMiddleware
+from registry import *
+from registry.db_registry import DbRegistry
+from registry.models import EntityType
+
+rp = "/"
+try:
+    rp = os.environ["API_BASE"]
+    if rp[0] != '/':
+        rp = '/' + rp
+except:
+    pass
+print("Using API BASE: ", rp)
```

Comment:
Use logger rather than print?

Reply:
Agreed.
This program doesn't have enough logs, I plan to add them.
Do we need it in this PR or later?